Cargo.toml

```diff
@@ -33,23 +33,28 @@ exclude = [
 [badges]
 maintenance = { status = "actively-developed" }
 
+[features]
+default = ["native-tls"]
```

Comment:
so to clarify here, `native-tls` refers to compiling krustlet with native OpenSSL. `rustls-tls` refers to the "native" (<-- where I get a little confused) Rust implementation of OpenSSL. Is that correct?

If we plan to support both features, should we document the two features in the developer docs? That might be helpful for other authors writing external providers.

Reply:
Native-tls matches the naming of the rust crate a features from other dependencies. So you are correct in that it uses native TLS implementations (whether that is openssl or something else).

Right now, this doesn't change anything for people working on the project. It still builds exactly as it did before (the default being native-tls). I was going to hold off on documentation until we got the TLS bootstrapping in and write it along with the documentation and pipeline for Windows builds. This is mainly to avoid writing a bunch of "when we do windows builds" kind of comments. If you'd think we really need the docs right now, I can definitely add those in a follow up

Reply:
Thanks for the clarification. Let's hold off until TLS boostrapping is in ans we can re-assess how we want to address this in the documentation (as most may require changes after TLS bootstrapping lands, anyways).